src/route/routes.js

```diff
@@ -7,10 +7,10 @@ export const routes = {
         routerPath: '/search',
         getPathWithParams: (id = '') => `/search?id=${id}`
     },
-    // error404: {
-    //     routerPath: '*',
-    //     getPathWithParams: () => '/error404'
-    // }
+    error404: {
+        routerPath: '*',
+        getPathWithParams: () => '*'
```

Comment:
Change this:
```
12: getPathWithParams: () => '*'
```

back to this:
```
12: getPathWithParams: () => '/error404'
```

Keep line 11 as it is.

Besides that looks good to me! Great solution :)

Reply:
Done